front/src/components/logo/logo.component.css

```diff
@@ -1,7 +1,7 @@
 div.logo { 
   font-style: normal;
   font-weight: 600;
-  font-size: 21px;    
+  font-size: 21px;     
```

Comment:
please, check your IDE settings or linter.
it is usefulness for fit history to have as changes  only white spaces